src/instructions-retired-explorer/instructions-retired-explorer.cs

```diff
@@ -476,6 +483,75 @@ static void GroupMethods()
             }
         }
 
+        static void FixR2RLengths()
+        {
+            List<ImageInfo> r2rMethodImages = new List<ImageInfo>();
+            List<ImageInfo> r2rAssembyImages = new List<ImageInfo>();
+            foreach (ImageInfo i in ImageMap.Values)
+            {
+                if (i.IsJitGeneratedCode && i.Tier == OptimizationTier.ReadyToRun && i.Size == 1)
+                {
+                    r2rMethodImages.Add(i);
+                }
+                else
+                {
+                    r2rAssembyImages.Add(i);
+                }
+            }
+
+            ImageInfo[] methodArray = new ImageInfo[r2rMethodImages.Count];
```

Comment:
nit: Can we just use List.ToArray for this? Or is there a specific reason why it needs to be this way